libmamba/include/mamba/core/link.hpp

```diff
@@ -23,6 +23,16 @@
 namespace mamba
 {
     std::string replace_long_shebang(const std::string& shebang);
+    std::string python_shebang(const std::string& python_exe);
+
+    static const std::regex shebang_regex(
+        "^(#!"                      // pretty much the whole match string
+        "(?:[ ]*)"                  // allow spaces between #! and beginning of the executable path
+        "(/(?:\\\\ |[^ \n\r\t])*)"  // the executable is the next text block without an escaped
```

Comment:
previously we had a bug with no doubly escaped `\\`

Reply:
We shoudl avoid global variables in headers and prefer functions to get them (to avoid a possible static initialization order fiasco).